utils/zerovec/benches/zerovec_serde.rs

```diff
@@ -86,27 +92,22 @@ fn u32_benches(c: &mut Criterion) {
 }
 
 #[cfg(feature = "bench")]
-fn char_benches(c: &mut Criterion) {
-    const ORIGINAL_CHARS: &[char] = &[
-        'ⶢ', '⺇', 'Ⱜ', '◁', '◩', '⌂', '⼅', '⏻', '⢜', '◊', 'ⲫ', '⏷', '◢', '⟉', '℞',
-    ];
-
+fn char_benches(c: &mut Criterion, original_chars: &[char], char_zero_vec: &ZeroVec<char>) {
     c.bench_function("zerovec_serde/serialize/char/slice", |b| {
-        b.iter(|| bincode::serialize(&Vec::from(ORIGINAL_CHARS)));
+        b.iter(|| bincode::serialize(&Vec::from(original_chars)));
```

Comment:
Nit, here and below: we should be `black_box`ing all external variables. It's a mistake that we did it in some of the benches but not others.